packages/hub/indexers.js

```diff
@@ -183,6 +187,7 @@ class Indexers {
     } finally {
       running.destroy();
     }
+    this.events.emit('index_update');
```

Comment:
I think it might be a good idea to include the hints in the event that is emitted so that subscribers to this event can make more nuanced decisions about what to do based on what is being indexed.

Reply:
I think we can do one better. Instead of this coarse event, we can emit events whenever an indexer calls the `save` or `delete` operations.